src/connectors/KafkaConnector.ts

```diff
@@ -20,7 +20,6 @@ export class KafkaConnector implements IConnector {
             messages: [{
                 value: config.value
```

Comment:
Why are removing these snappy compressions? Is being configured already somewhere?